frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -217,7 +216,18 @@ export const createDeployment = (
 
   const imageStreamName = imageStream && imageStream.metadata.name;
   const defaultLabels = getAppLabels(name, application, imageStreamName, tag);
-  const defaultAnnotations = getAppAnnotations(repository, ref);
+  const annotations = {
+    ...getAppAnnotations(repository, ref),
+    ...{
```

Comment:
Same here, no need to spread, just define them directly after the `getAppAnnotations` spread

Reply:
done